chart/values.yaml

```diff
@@ -87,6 +87,7 @@ injector:
     repo: chaos-injector
   annotations: {} # extra annotations passed to the chaos injector pods
   labels: {} # extra labels passed to the chaos injector pods
+  tolerations: [] # extra tolerations passed to the chaos injector pods
```

Comment:
Could you add a full commented example toleration? Much like `allowedHosts` or `kubeDNS` keys just below this? Otherwise a user will only know the possible options from reading the code

Reply:
Updated!